### DISCUSSION TITLE
Certify Forecast Visual

### DISCUSSION THREAD

**Discussion by jthornton2** [nbrown02/FlowViz#129] [Q&A] [answered]
Currently, my organization will not permit the use of uncertified visuals with Power BI. The FlowViz Forecast visualization offers tremendous potential value to my teams, however, I am unable to use it do to the lack of certification.

Do you plan to certify the Forecast visual with the AppSource?

**Comment by nbrown02** [comment]
It's not my visual so unfortunately it's down to the creator of it to get it certified :(

**Comment by jthornton2** [comment]
Oh, who is the creator?

**Reply by nbrown02** [reply]
https://github.com/liprec/powerbi-d3jsvisual

**Reply by jthornton2** [reply]
Got it, thank you!

BTW, I am very impressed with FlowViz and we are using it to help our development teams. Great work!

**Reply by nbrown02** [reply]
Thank you! Please leave a review in the ADO Marketplace if you get time :)

**Reply by nbrown02** [reply, marked as answer]
@jthornton2 in the latest version I have changed the forecast visual to be a native visual bar chart rather than a d3 visual. This means the certification issue will have gone away :)
